bundle/src/test/java/com/adobe/acs/commons/mcp/impl/TestGenericReportExcelServlet.java

```diff
@@ -39,7 +43,15 @@
 public class TestGenericReportExcelServlet {
     @Rule
     public final SlingContext slingContext = new SlingContext(ResourceResolverType.RESOURCERESOLVER_MOCK);
-    
+
+    @Before
+    public void setUp(){
+        slingContext.registerService(Injector.class, new ValueMapInjector());

```

Comment:
This should be registered by default: https://github.com/apache/sling-org-apache-sling-testing-sling-mock/blob/a223de695f7ce3b55e51767ac58874a0fac08adf/core/src/main/java/org/apache/sling/testing/mock/sling/context/SlingContextImpl.java#L169-L183, same for ChildResourceInjector.

Reply:
A good catch. I thought that the test was failing because these were missing, but it turned out to be ValueMapInjector 
 instead of ChildResourceInjector .